_posts/2015-10-30-october-newsletter.md

```diff
@@ -0,0 +1,292 @@
+
+### Bountysource salt campaign
+
+For those who don't know yet, Bountysource launched a new platform that allows
+open source projects to obtain sustainable crowdfunding. This platform is
+conveniently called ["salt"][history-of-salt], and Neovim was one of the first
+projects to use it.
+
+The [first campaign][first-campaign] was very
+successful and raised about $35,000, which allowed me to work full-time on
+Neovim for roughly 6 months. Being very enthusiastic about the project and
+unable to meet all goals in those months, I continued to dedicate a very
+significant portion of my time to Neovim, so much that it started hurting my
+personal and professional life. This continued until February when I saw that I
+simply couldn't continue with my old pace. Around that time that @rappo offered
+me to test the salt platform beta version and I saw it as a way to continue my
+work on Neovim.
+
+Like it's predecessor, [the salt campaign][salt-campaign] was very successful
+and allowed me to continue Neovim contributions (in a healthy way) for the past 6
+months, thank you!
+
+### Building Neovim from source
+
+Did we ever mention how easy it is to build and install Neovim from source?
+While it has a good number of dependencies, the build system automatically
+downloads and builds everything without cluttering your system. Check out
+the [installation page on the Neovim Wiki][installing-neovim-source] for the
+exact steps.
+
+## Development News
+
+### XDG Support
+
+Neovim now follows the [XDG directory specification][xdg-spec]. This was
+[proposed][xdg-proposal] by @ZyX-I when the project started, but only a couple
+of months ago we received a [PR][xdg-pr1] from @Yamakaky which was superseded by
+@jck in a [later PR][xdg-pr2] and again by @ZyX-I in a [final PR][xdg-pr3] that
+was merged recently.
+
+Since following the XDG directory specification, Neovim now looks for user
+configuration files such as `.nvimrc` and those under `~/.nvim` in the
+`~/.config` directory, which can be overriden by the `$XDG_CONFIG_HOME`
+environment variable. The specification also states that cache files should be
+stored in a separate directory (`~/.local/share`), which is where files like
+viminfo (now ShaDa) or backup/swap can optionally go.
+
+This change makes it simpler for users to backup and manage their configuration
+since it will be stored with other programs that also follow the specification,
+not to mention it keeps the home directory cleaner.
+
+Step by step instructions on how to migrate existing configuration can be found
+at [:h nvim-from-vim][nvim-from-vim].
+
+### ShaDa (Shared Data)
+
+@ZyX-I [major ShaDa PR][shada-pr] was merged. It completely replaced the viminfo
+file for storing user data such as register contents, command history,
+variables, jump list and so on.
+
+A [known problem][shada-proposal] with viminfo is that two Vim instances running
+concurrently will override each other's data. ShaDa is a new storage format
```

Comment:
@Pyrohh No, override. Except for the history and marks, whichever Vim exits last wins. Though for history under some circumstances you may take it as clobbering, I personally do not: without timestamps history merging is not perfect, but it works well enough.